src/main/kotlin/no/nav/familie/ef/søknad/api/RegistrerArbeidsaktivitetController.kt

```diff
@@ -19,16 +19,16 @@ import org.springframework.web.bind.annotation.RestController
 @RestController
 @RequestMapping(path = ["/api/registrerarbeid"], produces = [APPLICATION_JSON_VALUE])
 @ProtectedWithClaims(issuer = InnloggingUtils.ISSUER, claimMap = ["acr=Level4"])
-class RegistrerArbeidsaktivitetController(val skjemaService: SkjemaService, val featureToggleService: FeatureToggleService) {
+class RegistrerArbeidsaktivitetController(val skjemaService: SkjemaService, val featureToggleService: FeatureToggleService,  private val oppslagService: OppslagService) {
 
     @PostMapping
     @Protected
     fun sendRegistrerArbeidsAktivitet(@RequestBody arbeidssøker: Arbeidssøker): Kvittering {
-        val valueAsString = objectMapper.writeValueAsString(arbeidssøker)
-        val arbeidssøker: Arbeidssøker = objectMapper.readValue(valueAsString, Arbeidssøker::class.java);
-        val fnrFraToken = InnloggingUtils.hentFnrFraToken()
+
         return featureToggleService.enabledEllersHttp403("familie.ef.soknad.registrerarbeidssoker") {
-            skjemaService.sendInn(arbeidssøker, fnrFraToken)
+            val fnrFraToken = InnloggingUtils.hentFnrFraToken()
+            val forkortetNavn = oppslagService.hentSøkerinfo().søker.forkortetNavn
+            skjemaService.sendInn(arbeidssøker, fnrFraToken, forkortetNavn)
```

Comment:
HVIS dette tryner for kode6/7 - kan vi isåfall catche feilen og wrappe navnet til å være blank? 

Prøver å opprette en kode6-person nå - så ser vi hva som skjer :D

Reply:
Tror dette skal fungere fint; laget testpersonen 03118922157 (Absurd Bamse) som kode6 - du kan teste med den og se hva som skjer.